microsoft-edge/webview2/concepts/threading-model.md

```diff
@@ -32,6 +32,52 @@ The only exception is for the `Content` property of `CoreWebView2WebResourceRequ
 
 Callbacks including event handlers and completion handlers run serially.  
 After you run an event handler and begin a message loop, you're unable to run any event handler or completion callback in a re-entrant manner.  
+If you use methods that will spin up nested message loops such as `ShowDialog` synchronously within a WebView event handler, it will lead to reentrancy and leave the event handler in stack indefinitely.
```

Comment:
Are there other methods that do this other than ShowDialog? We should try and provide customers the list, or ways to know if the method will do so.

Reply:
@Reezaali - are we allowed to use 'spin up'?

Reply:
The style guides don't cover "spin up".  We suggest "create" instead, if that's technically correct & clear.